src/App.js

```diff
@@ -1,29 +1,14 @@
-import logo from './logo.svg';
+// import logo from './logo.svg';
```

Comment:
- Kindly remove any unnecessary lines of code to make your code base good and neat 😉